test/specs/palindrom.errors.test.js

```diff
@@ -211,122 +211,125 @@ describe('Palindrom', () => {
             });
         });
         context('Numbers Validation', function() {
-            it('Initial HTTP response: out of range numbers should call onIncomingPatchValidationError event with a RangeError', async () => {
-                fetchMock.mock(getTestURL('testURL'), {
+            const remoteUrl = getTestURL('testURL');
+            beforeEach(() => {
+                fetchMock.mock(remoteUrl, {
                     status: 200,
                     body: `{"value": ${Number.MAX_SAFE_INTEGER + 1}}`
```

Comment:
Moving mock server declaration to `beforeEach` caused some confusion between properties `val` and `value` used in some tests. Many tests still do replace operations on `/val`, simply using the fact that replace acts like add if a property does not exist. I find it not clean.

Maybe let's rename whatever is called `value` and `val` to something more distinct, like `amount`